vespajlib/src/main/java/com/yahoo/slime/JsonFormat.java

```diff
@@ -17,26 +22,31 @@
 public final class JsonFormat implements SlimeFormat {
 
     private final static byte [] HEX = Utf8.toBytes("0123456789ABCDEF");
-    private final boolean compact;
+    private final int indent;
+
+    public JsonFormat(int indent) {
+        this.indent = requireInRange(indent, "JSON space indent count", 0, 8);
+    }
+
     public JsonFormat(boolean compact) {
-        this.compact = compact;
+        this(compact ? 0 : 1);
```

Comment:
Change this to be the same as in `SlimeUtils` (2)?

Reply:
Maybe. 1 is just a bad choice. Didn't dare do it in fear o breaking test data, but then again I already changed empty arrays/objects to render on one line, so perhaps ...